### ISSUE TITLE
problems on ubuntu10.04

### ISSUE THREAD

**Issue by rechardchen** [doug/depthjs#20]
I build the chrome extension under chrome-extention-mac on ubuntu, 
but when running in chromium-browser, i just find the plugin is not stable and sometimes can not start properly.
have you tested the extension on linux?

btw, i tried to build the npapi plugin on mac with build-mac.sh, but failed...

**Comment by kench** [comment]
Have you tried the build-linux.sh script in chrome-extension-mac?
The basic demo doesn't receive events on Linux yet (possibly because of issue #19), but the Kinect camera does initialize/tilt when I load the demo.

I'll see following the directions listed in issue #19 solves the problem.  I'll try it out this weekend.

**Comment by rechardchen** [comment]
yes, i just run the build-linux script under chrome-extension-mac/plugin;
i tried the solution in issue #19 , but still it did not work :(

2011/7/1 kench <
reply@reply.github.com>

> Have you tried the build-linux.sh script in chrome-extension-mac?
> The basic demo doesn't receive events on Linux yet (possibly because of
> issue #19), but the Kinect camera does initialize/tilt when I load the demo.
> 
> I'll see following the directions listed in issue #19 solves the problem.
>  I'll try it out this weekend.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/doug/depthjs/issues/20#issuecomment-1480357

## 

/*----------------------------------
   陈虎    Rechardchen

----------------------------------*/

**Comment by azinman** [comment]
A patch was submitted to help it run on linux.

What failed? It seems that we need to update some APIs for the changes in libfreenect. Hopefully my colleague can do that while I finish my dissertation.

Aaron

## 

aaron zinman :: mit media lab :: web.media.mit.edu/~azinman (http://web.media.mit.edu/%7Eazinman) :: protocol.by/azinman (http://protocol.by/azinman) :: courteous.ly/mhuoL3 (http://courteous.ly/mhuoL3)

**Comment by kench** [comment]
> A patch was submitted to help it run on linux.

I built the NPAPI plugin, but never modified manifest.json to load it.  I "patched" it to make it work.
